Tasks/VsTest/taskinputparser.ts

```diff
@@ -34,6 +34,14 @@ export function getDistributedTestConfigurations() {
     }
     tl._writeLine(tl.loc('dtaNumberOfAgents', dtaConfiguration.numberOfAgentsInPhase));
 
+    let useVsTestConsole = tl.getVariable('UseVsTestConsole');
+    if (useVsTestConsole) {
+        dtaConfiguration.useVsTestConsole = useVsTestConsole;
+    }
+    else {
+        dtaConfiguration.useVsTestConsole = 'true';
```

Comment:
nit: initialize with 'true' as default value so that you dont need else condition